internal/controller/linodemachine_controller.go

```diff
@@ -513,6 +513,14 @@ func (r *LinodeMachineReconciler) reconcileUpdate(ctx context.Context, logger lo
 		machineScope.LinodeMachine.Status.Ready = true
 		conditions.MarkTrue(machineScope.LinodeMachine, clusterv1.ReadyCondition)
 	}
+
+	// Clean up after instance creation.
+	if linodeInstance.Status == linodego.InstanceRunning && machineScope.Machine.Status.Phase == "Running" {
+		if err := deleteBootstrapData(ctx, machineScope); err != nil {
+			logger.Error(err, "Fail to bootstrap data")
```

Comment:
I think we might want to have deleteBootstrapData function log the deletion error. We might also not want to log it as an error if we're not doing anything with that error like requeuing to retry the deletion (though we might want to do that in the case we get some kind of error like rate-limiting?)